packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -375,15 +375,15 @@ const SearchInputBase: React.FunctionComponent<SearchInputProps> = ({
         </InputGroupItem>
       )}
       {!!onSearch && (
-        <InputGroupItem isPlain>
+        <InputGroupItem>
           <Button
             type="submit"
             variant={ButtonVariant.control}
             aria-label={submitSearchButtonLabel}
             onClick={onSearchHandler}
             isDisabled={isDisabled}
           >
-            <ArrowRightIcon />
```

Comment:
Should this be updated no what Eric's PR  #9609 went in

Reply:
I like that, what do you think @thatblindgeye and @wise-king-sullyman? Instead of adding the class directly to an icon, use the icon component. It's not hard-coding a class, and if we change the classname or how an icon mirrors/flips, it's an update to the icon component instead of potentially having to update all components that are referencing this class directly.

Reply:
I'd be in favor of that. I have a similar situation in my DataList PR. It'd also mean using our Icon component in more places as well so could be win-win.